Install ibmz #11022 keichwa (#193)
uyuni-project/uyuni-docs: modules/installation/pages/install-ibmz.adoc

It assumes that you are a z/VM systems programmer trained on {ibmz} operating protocols, and steps you through installing {productname} onto an existing mainframe system.
This section does not cover the variety of hardware configuration profiles available on {ibmz}, but provides a foundational overview of the procedure and requirements necessary for a successful {productname} Server deployment on {ibmz}.

> This section describes how to install {susemgr} Server from the {sle} installation media.

Reviewer: ```suggestion
This section describes how to install {susemgr} Server using {sle} installation media.
```

> For this topic we assume that you have already registered your {susemgr} product with {scc} and have obtained a registration code.

Reviewer: ```suggestion
You must have already registered your {susemgr} product with {scc}, and have obtained a registration code.
```

.Media Requirements:

For media requirements, see xref:installation:general-requirements[].

== Install {productname} on {ibmz}

This section covers the installation of {productname} as a product of the {sle} family.

> For general information on deploying a product on IBM Z hardware, see https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-zseries.html.

Reviewer: ```suggestion
For general information about deploying a product on {ibmz} hardware, see https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-zseries.html.
```
